app/controllers/collections_controller.rb

```diff
@@ -3,11 +3,7 @@ class CollectionsController < ManageCollectionsController
   include CMRCollectionsHelper
 
   before_action :set_collection
-  before_action :ensure_correct_collection_provider, only: [:edit, :clone, :revert, :destroy]
-
-  skip_before_action :proposal_mode_enabled?, only: [:show]
-  # the only functionality that should be available in Draft MMT should be
-  # to view the collection preview
+  before_action :ensure_correct_collection_provider, only: %i[edit clone revert destroy]
```

Comment:
I know rubocop was complaining to change this to use `%i`, but I think we made a decision a while ago to keep before_actions without that, because that is how Rails has been doing it for quite a while.